packages/core/src/tracing/transaction.ts

```diff
@@ -264,11 +264,13 @@ export class Transaction extends SpanClass implements TransactionInterface {
       dsc.transaction = this.name;
     }
 
+    if (this.sampled !== undefined) {
+      dsc.sampled = String(this.sampled);
+    }
+
     // Uncomment if we want to make DSC immutable
     // this._frozenDynamicSamplingContext = dsc;
 
-    client.emit && client.emit('createDsc', dsc);
```

Comment:
Removed this because `getDynamicSamplingContextFromClient` already emits `createDsc` on the client as a side effect, so no need to duplicate it.